elytron/src/main/java/org/wildfly/extension/elytron/ElytronDefinition.java

```diff
@@ -364,6 +365,9 @@ protected void revertUpdateToRuntime(OperationContext context, ModelNode operati
     @Override
     public void registerAdditionalRuntimePackages(ManagementResourceRegistration resourceRegistration) {
         resourceRegistration.registerAdditionalRuntimePackages(RuntimePackageDependency.required("org.wildfly.security.elytron"));
+        if (resourceRegistration.getStability().enables(Stability.COMMUNITY)) {
+            resourceRegistration.registerAdditionalRuntimePackages(RuntimePackageDependency.required("org.wildfly.security.elytron-dynamic-ssl"));
+        }
```

Comment:
@Skyllarr Why is this here? This seems like something the ResourceDefinition returned from SSLDefinitions.getDynamicClientSSLContextDefinition() should handle. And there'd be no conditional logic as the availability of that ResourceDefinition is already conditional on the stability setting.

@jfdenise I think we should discuss this in zulip, not here, but I'm interested in your opinion re this kind of logic in the general sense, i.e. in cases where the relevant RD is always present but the need for a particular additional runtime package depends on the stability level. I *think* it's ok. The module would only be depended upon optionally, so something is needed to tell Galleon to provision it. We don't want to always tell Galleon to provision it, because the module.xml will have a stability setting, and if we always tell Galleon to provision it in RD. registerAdditionalRuntimePackages, but then the module.xml indicates that Galleon should not provision it, we have a conflict.

Reply:
@bstansberry Thanks, I moved the package registration to the registerAdditionalRuntimePackages method so it is registered when getDynamicClientSSLContextDefinition is invoked